vms/administrator/tests/test_data.json

```diff
@@ -0,0 +1,282 @@
+[
+  {
+    "volunteer": [
+      "uname1",
```

Comment:
this representation is a start, but only half done. You need it like this. 
```
"volunteer":  {
     "first_name":  "uname1",
      "last_name": "uname1", 
....
}

and then use it as: 

v1 = Volunteer(
+        email=volunteer['email'],
+        first_name=volunteer['first_name'],
+        last_name=volunteer['last_name'],
+        address=volunteer[5],
+        city=volunteer[6],
+        state=volunteer[7],
+        country=volunteer[8],
+        phone_number=volunteer[9],
+        user=u1
+    )

Reply:
Umm... if I change this then I'd have to change the structure we earlier discussed (lists -> dict), I think this can be a part of the new issue we'll create to change use of lists to dict.

Reply:
okey then. Once you link that new issue here, I will finish my review :)

Reply:
here you go @tonythomas01 https://github.com/systers/vms/issues/697

Reply:
Looks good then. approving.

Reply:
I agree with @tonythomas01 . Will this issue be taken up by you, @Monal5031 ?

Reply:
Not atm, but if time persists before I move on to postorius I'll do it.